project-base/storefront/utils/animations/framerMotionPlugins.ts

```diff
@@ -0,0 +1,4 @@
+// used for tree shaking (https://www.framer.com/motion/guide-reduce-bundle-size/#lazy-loading)
+import { domAnimation } from 'framer-motion';
+
+export default domAnimation;
```

Comment:
Can we somehow measure if this works? Looks pretty _sus_ 😆

Reply:
it comes from documentation, but I will try to check it with bundle analyzer 
https://www.framer.com/motion/guide-reduce-bundle-size/

Reply:
Mostly it won't affect any metrics, the library is 10Kb so even if it doesn't work, it won't have any significant consequences